components/sysutils/bat/results-all.master

```diff
@@ -0,0 +1,533 @@
+   Compiling autocfg v1.1.0
```

Comment:
This should go into `test` subdirectory.

Also, it looks like you added wrong file here.  Is this a copy of `build/test/amd64/results-64.snapshot`?

Reply:
That seems to be the file needed so it doesn't show differences.

Reply:
@mtelka Yes I copied the file you mentioned. Unfortunately the line output does not seem to be stable and there are some compilation results that end in the test output due to the way our framework captures the output. Can you elaborate what you do not like about the results file?